update.js

```diff
@@ -1,30 +1,30 @@
 #!/usr/bin/env node
 
-'use strict'
-
-const fs = require('fs')
-const util = require('util')
-const path = require('path')
-const semver = require('semver')
-const ethUtil = require('ethereumjs-util')
-const ipfsImporter = require('ipfs-unixfs-importer')
-const IPLD = require('ipld')
-const inMemory = require('ipld-in-memory')
-const swarmhash = require('swarmhash')
-
-// This script updates the index files list.js and list.txt in the directories containing binaries,
-// as well as the 'latest' and 'nightly' symlinks/files.
-
-const ipfsHash = async (content) => {
-  const iterator = ipfsImporter.importer([{ content }], await inMemory(IPLD), { onlyHash: true })
-  const { value, done } = await iterator.next()
-  if (done) {
-    throw new Error('Failed to calculate an IPFS hash.')
-  }
-
-  await iterator.return()
-  return value.cid.toString()
-}
+//this is a workaround for the semver as far as it is a CommonJS module
+import semver from "semver"
+const { compare, gt } = semver
```

Comment:
Not sure if I follow what the workaround is here. Why not do like the following?
```javascript
import { compare, gt } from "semver";
```

Reply:
because `The requested module 'semver' is a CommonJS module, which may not support all module.exports as named exports.` I added it into the comment

Reply:
I left it as `import semver from "semver"` with the further usage via `semver.compare` and `semver.gt`

Reply:
Hum, ok then. I thought `semver` supported both types.